src/popup/pages/Retip.vue

```diff
@@ -140,7 +135,12 @@ export default defineComponent({
           : AETERNITY_COIN_PRECISION,
       ).toNumber();
       loading.value = true;
-      await watchUntilTruthy(() => tippingV1.value);
+      const { tippingV1, tippingV2 } = await getTippingContracts();
+      const tippingContract = computed(
+        () => tipId.includes('_v2') || tipId.includes('_v3')
```

Comment:
A suggestion for this one as I don't like if something repeats one after another
```suggestion
        () => ['_v2', '_v3'].some((phrase) => tipId.includes(phrase))
```